dialog-polyfill.js

```diff
@@ -115,6 +121,26 @@
     },
 
     /**
+     * Maybe assign a default role attribute to this dialog that is being
+     * polyfilled. Without one, the dialog won't have a useful a11y tree generated
+     * for its core usage in a Web application.
+     */
+    maybeSetADefaultRoleValue: function(){
+      if ( !this.dialog.hasAttribute('role') || this.dialog.getAttribute('role') === ''  ) {
+        this.dialog.setAttribute( 'role', 'dialog'  );
```

Comment:
So this approach will just eventually set the `role="dialog"` on each dialog. It will never transition back to not being a dialog.

With that in mind, I'd like to suggest #95 as just a quick fix since it's basically paramount to this approach :)